technic/machines/register/generator.lua

```diff
@@ -146,7 +159,7 @@ function technic.register_generator(data)
 						}
 					)..pipeworks.button_label
 			end
-			meta:set_string("formspec", generator_formspec..form_buttons)
+			update_generator_formspec(meta, desc, percent, form_buttons)
```

Comment:
Here percent should prob be just 0
```suggestion
			update_generator_formspec(meta, desc, 0, form_buttons)
```